spec/index.html

```diff
@@ -248,5 +249,155 @@ <h2>Basic Concepts</h2>
+              it might therefore be necessary to convert between IRIs and URIs as specified in
+              <a data-cite="RFC3986#section-5.1">section 3 Relationship between IRIs and URIs</a>
+              of [[RFC3987]].</p>
+          </dd>
+        </dl>
+      </dd>
+      <dt>Encoding considerations:</dt>
+      <dd>See <a data-cite="RFC8259#section-11">RFC&nbsp;8259, section 11</a>.</dd>
+      <dt id="iana-security">Security considerations:</dt>
+      <dd>See <a data-cite="RFC8259#section-12">RFC&nbsp;8259, section 12</a> [[RFC8259]]
+        <p>When processing YAML-LD documents, links to remote contexts and frames are
+          typically followed automatically, resulting in the transfer of files
+          without the explicit request of the user for each one. If remote
+          contexts are served by third parties, it may allow them to gather
+          usage patterns or similar information leading to privacy concerns.
+          Specific implementations, such as the API defined in the
+          JSON-LD 1.1 Processing Algorithms and API specification [[JSON-LD11-API]],
+          may provide fine-grained mechanisms to control this behavior.</p>
+        <p>YAML-LD contexts that are loaded from the Web over non-secure connections,
+          such as HTTP, run the risk of being altered by an attacker such that
+          they may modify the YAML-LD <a data-cite="json-ld11#dfn-active-context">active context</a> in a way that
+          could compromise security. It is advised that any application that
+          depends on a remote context for mission critical purposes vet and
+          cache the remote context before allowing the system to use it.</p>
+        <p>Given that YAML-LD allows the substitution of long IRIs with short terms,
+          YAML-LD documents may expand considerably when processed and, in the worst case,
+          the resulting data might consume all of the recipient's resources. Applications
+          should treat any data with due skepticism.</p>
+        <p>As YAML-LD places no limits on the IRI schemes that may be used,
+          and vocabulary-relative IRIs use string concatenation rather than
+          IRI resolution, it is possible to construct IRIs that may be
+          used maliciously, if dereferenced.</p>
+      </dd>
+      <dt>Interoperability considerations:</dt>
+      <dd>Not Applicable</dd>
+      <dt>Published specification:</dt>
+      <dd>http://www.w3.org/TR/yaml-ld</dd>
+      <dt>Applications that use this media type:</dt>
+      <dd>Any programming environment that requires the exchange of
+        directed graphs. Implementations of YAML-LD have been created for
+        <span class="note">FIXME</span>.
+      </dd>
+      <dt>Additional information:</dt>
+      <dd>
+        <dl>
+          <dt>Magic number(s):</dt>
+          <dd>Not Applicable</dd>
+          <dt>File extension(s):</dt>
+          <dd>.yamlld</dd>
```

Comment:
Q: for json-ld the unique extension is .jsonld, right?

Reply:
One might use `.yamld`, but consider this a placeholder subject to group concensus.

Reply:
yamlld LGTM